libraries/AC_Fence/AC_Fence.cpp

```diff
@@ -146,6 +148,13 @@ const AP_Param::GroupInfo AC_Fence::var_info[] = {
     // @User: Standard
     AP_GROUPINFO_FRAME("OPTIONS", 11, AC_Fence, _options, static_cast<uint16_t>(AC_FENCE_OPTIONS_DEFAULT), AP_PARAM_FRAME_PLANE | AP_PARAM_FRAME_COPTER | AP_PARAM_FRAME_TRICOPTER | AP_PARAM_FRAME_HELI),
 
+    // @Param{Plane, Copter}: NTF_FREQ
```

Comment:
we need to get rid of the _FRAME specific stuff in AC_Fence.cpp, poor old rovers are missing out on some options